### ISSUE TITLE
new term endothelial to mesenchymal transition

### ISSUE THREAD

**Issue by NancyCampbell** [geneontology/go-ontology#13939]
Hi,

May I request the new term cardiac endothelial to mesenchymal transition. 

A similar term cardiac epithelial to mesenchymal transition already exists GO:0060317.

synonym: EndoMT

Suggested definition (or better):
A transition where a cardiac endothelial cell loses apical/basolateral polarity, severs intercellular adhesive junctions, degrades basement membrane components and becomes a migratory mesenchymal cell
PMID 26053665 (figure 9 and relevant text - cardiac specific)
PMID 16162442 (cardiac specific)

Can you please add the following
GOC:BHF, GOC:nc

Thank you

Nancy

**Comment by pgaudet** [comment]
Hi @NancyCampbell 

I added the term as a child of 'endothelial cell differentiation'. 
I made this definition, also based on PMID:28181491 
"A transition where a cardiac endothelial cell loses apical/basolateral polarity, severs intercellular adhesive junctions, degrades basement membrane components and becomes a migratory mesenchymal cell. Endocardial cells (specialized endothelial cells that line the heart) undergo EndMT, and give rise to mesenchymal cells necessary for proper heart development. EndMT, specifically generates valve progenitor cells that give rise to the mitral and tricuspid valves. EndMT also contributes to endocardial cushion formation, as well as to generation of cardiac fibroblasts and smooth muscle cells, but not cardiac myocytes."

however I am unclear about the relation to other existing terms, for example: 
- GO:0003199 endocardial cushion to mesenchymal transition involved in heart valve formation
-  GO:0090500 endocardial cushion to mesenchymal transition
- GO:0060317 cardiac epithelial to mesenchymal transition


![image](https://user-images.githubusercontent.com/4782928/28419812-853678d8-6d60-11e7-9a59-551f334911c6.png)


Looking at this I am not clear about the connection between endothelial and endocardial cells. Let me know if this seems OK to you. 

Thanks, Pascale
